### PR TITLE
Allanx2000

### PR DESCRIPTION
**Pull request by allanx2000**
Start prototyping the post control and needed assets

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -26,6 +40,26 @@ public void start(Stage primaryStage) throws Exception{
         tt.setAutoReverse(true);
         tt.setCycleCount(99);
         //tt.play();
+
+        api.getRants(Sort.ALGO, 10, 0).thenAcceptAsync(results -> {
```
Comment (tahnik) [inline src/main/java/Main.java]: This is not the place to get the rants. In a controller maybe? Also, you should use thenAccept()

File: src/main/java/controllers/PostControl.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Created by Tahnik Mustasin on 19/02/2017.
+ */
+public class PostControl extends BorderPane{
```
Comment (tahnik) [inline src/main/java/controllers/PostControl.java]: This class looks like a model, not a controller. Also, why is extending borderpane? You should use an FXML instead and modify that via controllers.

**Review by tahnik — CHANGES_REQUESTED** [review]

File: src/main/java/controllers/PostControl.java (hunk 2 above)
Comment (allanx2000) [reply src/main/java/controllers/PostControl.java]: https://docs.oracle.com/javase/8/javafx/fxml-tutorial/custom_control.htm

I followed this to make it a reuseable control

File: src/main/java/Main.java (hunk 1 above)
Comment (allanx2000) [reply src/main/java/Main.java]: Ah OK for supply and then accept confused, thought then accept her to be async as well.

This was just for testing the PostControl and highlight the issue I'm having with the image

**Review by tahnik — APPROVED** [review]

**Pull request merged**